.pre-commit-config.yaml

```diff
@@ -1,7 +1,6 @@
 ci:
   autofix_prs: false
   autoupdate_schedule: 'quarterly'
```

Comment:
```suggestion
  autoupdate_schedule: 'quarterly'
  skip: [no-commit-to-branch]
```

Reply:
Good call, not sure why I didn't think of this when removing the other skip we had. This is much better and yes it is for local developers.